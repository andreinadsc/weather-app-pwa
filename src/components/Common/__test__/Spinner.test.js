import { render } from '@testing-library/react'
import Spinner from '../Spinner';

describe('Spinner Common Component', () => {
    test('should render the spinner correctly', () => {
        const { baseElement } = render(<Spinner />)

        expect(baseElement).toMatchSnapshot();
    });
});