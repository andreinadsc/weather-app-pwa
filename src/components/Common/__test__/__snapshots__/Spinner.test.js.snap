// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner Common Component should render the spinner correctly 1`] = `
<body>
  <div>
    <div
      class="spinner"
    />
  </div>
</body>
`;
